lighthouse-core/report/report-generator.js

```diff
@@ -132,6 +132,7 @@ class ReportGenerator {
       // XSS, define a renderer that only transforms links and code snippets.
       // All other markdown ad HTML is ignored.
       const renderer = new marked.Renderer();
+      renderer.br = () => '<br>';
```

Comment:
Ah ok. I have a PR coming that introduces `<pre>` to the markdown renderer. We could also use that for block level code snippets, removing the need for `<br>` and the css you have. WDYT?